clients/src/main/java/org/apache/kafka/common/utils/AppInfoParser.java

```diff
@@ -57,7 +56,7 @@ public static void registerAppInfo(String prefix, String id) {
             AppInfo mBean = new AppInfo();
             ManagementFactory.getPlatformMBeanServer().registerMBean(mBean, name);
         } catch (JMException e) {
-            log.warn("Error registering AppInfo mbean", e);
+            log.warn("Error registering AppInfo mbean: {}", e.getMessage());
```

Comment:
I agree. But `WARN` should still not contain a stack trace -- that's for `ERROR` IMHO.